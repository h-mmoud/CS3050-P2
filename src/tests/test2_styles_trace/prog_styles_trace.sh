#!/bin/bash
sh ../run.sh --trace ../examples/styles.fp