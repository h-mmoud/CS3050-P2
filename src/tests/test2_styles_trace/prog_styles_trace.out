Call: proposed(hastings,cynthia)
Call: write(hastings)
Fail: accepted(cynthia,hastings)
Redo: proposed(lawrence,cynthia)
Call: write(lawrence)
Call: accepted(cynthia,lawrence)

hastings
lawrence

X = lawrence, Y = cynthia